### ISSUE TITLE
Ios: Can't choose sync provider with VoiceOver screenreader on iPhone

### ISSUE THREAD

**Issue by DGartmann** [laurent22/joplin#7553]
## Steps to reproduce
    
 1.  Turn VoiceOver on under accessibility and vision.
2.  Open Joplin and go to the synchronization screen.
3. 

## Describe what you expected to happen

I would expect that I can move from one item in the list of options to the next with the VoiceOver cursor.

**Comment by personalizedrefrigerator** [comment]
Can reproduce! It seems to work with TalkBack on Android, however.

**Comment by DGartmann** [comment]
Thanks for confirming. Hoping you or someone else is willing to try to solve this issue. It would be much apreciated.

**Comment by personalizedrefrigerator** [comment]
It looks like I won't be able to work on this for a few days (~~[iOS simulators don't seem to have VoiceOver](https://developer.apple.com/forums/thread/83458)~~ **Edit**: They do, VoiceOver just needs to be enabled systemwide). After a pull request has been created and merged it should (I think) be a few days to a week or two before the [iOS pre-release](https://discourse.joplinapp.org/t/ios-pre-release-2-10-is-now-available-updated-29-dec/28848) is updated.

My plan is to try adjusting accessibility properties [in Dropdown.tsx](https://github.com/laurent22/joplin/blob/bae7ff0ca5a49dd1ba61bbf107305d548d1fb2cf/packages/app-mobile/components/Dropdown.tsx#L127) to get VoiceOver to recognize the `TouchableOpacity` components (the things in the dropdown) as focusable.

**Comment by DGartmann** [comment]
Great. Will be looking forward to testing the fix. I am also seeing the same behavior of VoiceOver in the menu for choosing the language of the app. Should I create a separate issue?

**Comment by personalizedrefrigerator** [comment]
> Great. Will be looking forward to testing the fix. I am also seeing the same behavior of VoiceOver in the menu for choosing the language of the app. Should I create a separate issue?

No. The issue should be with all dropdown menus in the app and the linked pull request should fix all of these issues.

If you encounter other issues (e.g. sidebar never closing, no way to submit the "search" input) please report them if they don't yet exist on GitHub.

Thank you for reporting this!
